CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## [3.1.0](https://github.com/commercetools/merchant-center-application-kit/compare/v3.0.1...v3.1.0) (2018-11-28)
+
+#### 💅 Type: Enhancement
+
+- `application-shell`
+  - [#143](https://github.com/commercetools/merchant-center-application-kit/pull/143) feat(test-utils): run tests agains actual ApolloProvider ([@pa3](https://github.com/pa3))
+  - [#144](https://github.com/commercetools/merchant-center-application-kit/pull/144) Fix double dependency bundling ([@tdeekens](https://github.com/tdeekens))
+
+#### 🚀 Type: New Feature
+
+- `application-shell`, `i18n`
+  - [#142](https://github.com/commercetools/merchant-center-application-kit/pull/142) feat(app-shell/navbar): add menu item for custom apps ([@emmenko](https://github.com/emmenko))
```

Comment:
Not sure if we want to include this as it’s still feauture toggled

Reply:
Sure. I think it's best to then just not add a label? Cause it's hard to now for somebody else than the one opening PRs judging if things should be in the change log :)